### PR TITLE
rgw: STS Lite documentation correction

### PR DESCRIPTION
**Pull request by TRYTOBE8TME**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by pritha-srivastava** [comment]
add signed-off by in the commit message - that should be your red hat email id

File: doc/radosgw/STSLite.rst [hunk 1]
```diff
@@ -37,14 +37,16 @@ Parameters:
...
 by GetSessionToken.
-The following is an example of attaching the policy to a user 'TESTER1'::
-
-    s3curl.pl --debug --id admin -- -s -v -X POST "http://localhost:8000/?Action=PutUserPolicy&PolicyName=Policy1&UserName=TESTER1&PolicyDocument=\{\"Version\":\"2012-10-17\",\"Statement\":\[\{\"Effect\":\"Deny\",\"Action\":\"s3:*\",\"Resource\":\[\"*\"\],\"Condition\":\{\"BoolIfExists\":\{\"sts:authentication\":\"false\"\}\}\},\{\"Effect\":\"Allow\",\"Action\":\"sts:GetSessionToken\",\"Resource\":\"*\",\"Condition\":\{\"BoolIfExists\":\{\"sts:authentication\":\"false\"\}\}\}\]\}&Version=2010-05-08"
 
 The user attaching the policy needs to have admin caps. For example::
 
     radosgw-admin caps add --uid="TESTER" --caps="user-policy=*"
 
+The following is an example of a policy which needs to be attached to a user 'TESTER1'::
```
Comment (pritha-srivastava) [inline doc/radosgw/STSLite.rst]: this should be - the following is the policy that needs ...

Comment (TRYTOBE8TME) [reply doc/radosgw/STSLite.rst]: Okay

**Comment by TRYTOBE8TME** [comment]
jenkins test make check

**Comment by TRYTOBE8TME** [comment]
jenkins retest this please

**Comment by TRYTOBE8TME** [comment]
jenkins render docs

**Pull request merged**